log4j-core/src/main/java/org/apache/logging/log4j/core/appender/SyslogAppender.java

```diff
@@ -300,6 +307,11 @@ public B setLoggerFields(final LoggerFields[] loggerFields) {
             this.loggerFields = loggerFields;
             return asBuilder();
         }
+
+		public B setTimestampPrecision(final String timestampPrecision) {
```

Comment:
This indentation looks messed up. Spaces, no tabs, could be the issue?

Reply:
I have fixed it with latest commit.